### PR TITLE
updates to add sponsor page and replacing use of roadmap with 'futures' …

### PR DESCRIPTION
**Pull request by subsy**
…in multiple places

### CODE REVIEW

File: content/contribute/_index.md [hunk 1]
```diff
@@ -28,5 +28,5 @@ Jenkins X relies heavily on the enthusiasm and participation of the open-source
 * If you'd like to [contribute to the code](https://jenkins-x.io/contribute/development/) then try browse the [current issues](https://github.com/jenkins-x/jx/issues).
   * we have marked issues [help wanted](https://github.com/jenkins-x/jx/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) or [good first issue](https://github.com/jenkins-x/jx/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) to save you hunting around too much
   * in particular we would love help on getting Jenkins X [working well on windows](https://github.com/jenkins-x/jx/issues?q=is%3Aopen+is%3Aissue+label%3Awindows) or the [integrations with cloud services, git providers and issues trackers](https://github.com/jenkins-x/jx/issues?q=is%3Aissue+is%3Aopen+label%3Aintegrations)
- * for more long term goals we've the [long term roadmap](https://jenkins-x.io/contribute/roadmap)
+ * for more long term goals see the [Jenkins X Futures](https://jenkins-x.io/contribute/roadmap) page
```
Comment (jtnord) [inline content/contribute/_index.md]: why not change the URL and file names (given it has been aliased)?

Comment (subsy) [reply content/contribute/_index.md]: @jtnord only had v. limited time to make the change earlier and was playing safe ref external references - likely should add /contribute/futures and have /contribute/roadmap redirect, or just replace 
@rawlingsj @jtnord thoughts?

Comment (jtnord) [reply content/contribute/_index.md]: you already have the aliases configured so renaming the files and changing the href links for good measure should cover it IIUC.  YMMV - can be tested locally with `hugo server`

**Pull request merged**